pkgs/applications/virtualization/anbox/default.nix

```diff
@@ -25,7 +25,7 @@ let
 
   process-cpp = stdenv.mkDerivation rec {
     name = "process-cpp-${rev}";
-    rev = "3.0.1+16.10.20160616-0ubuntu1";
+    rev = "2.0.0+14.10.20140718-0ubuntu1";
```

Comment:
it seems that ```fetchbzr``` ignores ```rev``` and any random text written in ```rev``` results in the same output (note that ```sha256``` remains the same).

There is ```3.0.1``` mentioned https://code.launchpad.net/process-cpp and the Ubuntu package named ```libprocess-cpp3``` but how to fetch it?